nrf_modem/doc/dectphy.rst

```diff
@@ -0,0 +1,296 @@
+Additionally, it reject further scheduling of radio operations with the :c:enum:`nrf_modem_dect_phy_err.NRF_MODEM_DECT_PHY_ERR_TEMP_HIGH` error.
+
+In this event, the application must de-initialize the Modem library PHY interface (and thus the Modem firmware DECT NR+ PHY layer) by calling :c:func:`nrf_modem_dect_phy_deinit` and allow the device to cool.
+
+The application can then re-initialize the DECT PHY interface by calling :c:func:`nrf_modem_dect_phy_init`, and read the current measured temperature in the :c:member:`nrf_modem_dect_phy_callbacks.init` function parameters,
+to ensure the temperature has decreased below the allowed threshold.
+
+Physical layer capabilities
+===========================
+
+The application can retrieve the DECT NR+ modem firmware physical layer capabilities by calling the :c:func:`nrf_modem_dect_phy_capability_get` function.
+The list of supported capabilities is returned to the application in the :c:member:`nrf_modem_dect_phy_callbacks.capability_get` callback function.
+
+Radio operations
+****************
+
+The DECT PHY interface offers three radio operations: transmission (TX), reception (RX), and RSSI measurement.
+Each of these operations can be performed with different parameters.
+
+The completion of a radio operation is signalled to the application by the invocation of the :c:member:`nrf_modem_dect_phy_callbacks.op_complete` callback function, with the same handle that was specified by the application at the time the operation was scheduled.
+
+Transmission
+============
+
+The application can schedule a transmission by calling the :c:func:`nrf_modem_dect_phy_tx` function.
+
+The operation is asynchronous, and the successful completion of the :c:func:`nrf_modem_dect_phy_tx` function only signals that the request was sent to the modem.
+When the operation has completed, its result is signalled to the application in the :c:member:`nrf_modem_dect_phy_callbacks.op_complete` callback function.
+If any error has occurred in scheduling or executing the operation, it is returned in the callback.
+
+The operation has several parameters, including listen-before-talk (LBT) period and threshold.
+
+Channel frequency
+-----------------
+
+The DECT radio band is divided in different channels.
+The absolute radio channel frequency used for transmission is controlled by the :c:member:`nrf_modem_dect_phy_tx_params.carrier` field on the transmission parameters.
+
+.. important::
+   Ensure to abide to local regulations when transmitting!
+   To see which frequency channels can be used:
+
+   - In the European Union, you must refer to table 4.3.2.3-2 of `ETSI EN 301 406-2`_.
+   - In the USA, you must refer to `47 CFR Part 15.323`_.
+   - In all other territories, you must follow the local regulations.
+
+
+Length
+------
+
+Transmissions take time, that is, they have a length.
+
+A *transmission* length is expressed in sub-slots (or half slots). A transmission may take up to 15 sub-slots.
```

Comment:
A *transmission* length is expressed in slots or sub-slots (or half slots) as defined in the PHY header in tx parameters nrf_modem_dect_phy_hdr *phy_header